frontend/css/layout.css

```diff
@@ -64,25 +79,119 @@ li a {
     padding-top: var(--spacing-m);
 }
 
-.ideas {
-    text-align: center;
+.nav__icons__item {
+    flex-direction: column;
+    align-items: center;
+}
+
+.hamburger__btn-open {
+    margin-right: var(--spacing-m);
+}
+
+.nav__icons {
+    margin-right: var(--spacing-l);
+}
+
+@media (min-width: 992px) {
```

Comment:
Could the breakpoints be defined in `rem` or `em`? 
(If we want to close this PR though, we could document it in a separate issue.)